src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
```suggestion
    void draw();
```
<img width="968" alt="image" src="https://user-images.githubusercontent.com/68428976/178242883-77899746-55e5-4e9e-9f79-1f9c253f07db.png">